x-pack/platform/plugins/shared/task_manager/server/task_running/task_runner.ts

```diff
@@ -675,12 +676,18 @@ export class TaskManagerRunner implements TaskRunner {
           state,
           attempts = 0,
           shouldDeleteTask,
+          shouldDisableTask,
         }: SuccessfulRunResult & { attempts: number }) => {
           if (shouldDeleteTask) {
             // set the status to failed so task will get deleted
             return asOk({ status: TaskStatus.ShouldDelete });
           }
 
+          if (shouldDisableTask) {
+            shouldTaskBeDisabled = true;
+            return asOk({ status: TaskStatus.Idle });
```

Comment:
resetting the status to idle - debated trying to flag it as an error, but when we have zombie tasks, that's our fault, so there really isn't an error that we want to report ...